locales/locales.php

```diff
@@ -687,6 +687,15 @@ public function __construct() {
 		$es_co->slug = 'es-co';
 		$es_co->google_code = 'es';
 		$es_co->facebook_locale = 'es_CO';
+		
+		$es_CR = new GP_Locale();
+		$es_CR->english_name = 'Spanish (Costa Rica)';
+		$es_CR->native_name = 'Español';
+		$es_CR->lang_code_iso_639_1 = 'es';
+		$es_CR->lang_code_iso_639_3 = 'spa';
```

Comment:
`lang_code_iso_639_3` and `lang_code_iso_639_2` are the same for Spanish (https://en.wikipedia.org/wiki/Spanish_language), do you want both or just 2? And if just 2, why?